### ISSUE TITLE
selecting and drag&drop multiple tiles at once

### ISSUE THREAD

**Issue by andreasharbo** [arnaudframmery/qt-tile-layout#15]
This is a feature request :)

**Comment by arnaudframmery** [comment]
It seems to be an interesting feature :)

For a first version we can define that the user can select several tiles with a right click for example and then drag & drop them on another place on the layout.
It will require some work but it is feasible, I don't know when I will have time for that so don't expect to see this feature very soon ^^

**Comment by andreasharbo** [comment]
Yes, this a challenge :) i have implemented similar features without layouts, so i might have some ideas. I would suggest doing to like any other software would. Fx the File Explorer on Windows. Ctrl+LeftClick to select individual widgets, Shift+LeftClick to select a span :)

No rush, just putting ideas here so they don't get lost :)
